include/deal.II/hp/dof_handler.h

```diff
@@ -1120,6 +1220,22 @@ namespace hp
      */
     std::map<const cell_iterator, const unsigned int> coarsened_cells_fe_index;
 
+    /**
+     * Container to temporarily store the active_fe_index of every locally
+     * owned cell for transfer across distributed triangulations.
+     */
+    std::vector<unsigned int> active_fe_indices;
```

Comment:
Can you open a github issue that reminds us to put all of these temporary objects into some kind of structure? They all have perfectly innocuous names that suggest that they store some kind of important information, but in reality they are all only present and valid during very specific phases of the program. We should mark this somehow by providing them a separate home.

Not for this PR, but if you open a separate issue we can do this at a later time -- feel free to copy the comment above.